### PR TITLE
Implementing split dropdown menu only for users who are members of or…

### PR DESCRIPTION
**Pull request by adamjgrant**

### CODE REVIEW

**Comment by adamjgrant** [comment]
screen shot 2015-08-03 at 12 34 25 pm
screen shot 2015-08-03 at 12 34 34 pm

**Comment by adamjgrant** [comment]
For users not in orgs, there will be no change. If you are in an org, you'll use the split design where you can quickly jump to your profile or an org.

**Comment by adamjgrant** [comment]
@cvializ @camhashemi @andrewconner @ryanpbrewster

**Comment by andrewconner** [comment]
Works for me

File: kifi-backend/modules/shoebox/angular/src/common/classes.styl [hunk 1]
```diff
@@ -11,6 +11,21 @@
...
+  &.kf-dropdown-menu_split
+    display: flex
+    align-content: stretch
+    padding-top: 0
+    padding-bottom: 0
+
+    .kf-dropdown-menu-nav a
+      &:first-child
+      &:only-child
+        margin-top: 5px
```
Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/common/classes.styl:23]: I'm a little confused here. Do you need to match the `:only-child` selector? If there's only one element, isn't it both the first and last child anyway?

Comment (adamjgrant) [inline kifi-backend/modules/shoebox/angular/src/common/classes.styl:23]: Yes, I thought I was being clever here but I was mixing it up with something else.

Might have been a force of habit from implementations where `last-child` took precedence over `first-child`, but only if they have the same properties.

Comment (adamjgrant) [inline kifi-backend/modules/shoebox/angular/src/common/classes.styl:23]: Good catch.

**Comment by cvializ** [comment]
Late to the rating party, but this looks :+1:

**Pull request merged**